src/transformers/models/layoutlm/configuration_layoutlm.py

```diff
@@ -183,6 +183,17 @@ def generate_dummy_inputs(
             raise ValueError("Cannot generate dummy inputs without PyTorch installed.")
         import torch
 
+        # If dynamic axis (-1) we forward with a fixed dimension of 2 samples to avoid optimizations made by ONNX
+        batch_size = compute_effective_axis_dimension(
+            batch_size, fixed_dimension=OnnxConfig.DEFAULT_FIXED_BATCH, num_token_to_add=0
+        )
+
+        # If dynamic axis (-1) we forward with a fixed dimension of 8 tokens to avoid optimizations made by ONNX
+        token_to_add = tokenizer.num_special_tokens_to_add(is_pair)
+        seq_length = compute_effective_axis_dimension(
+            seq_length, fixed_dimension=OnnxConfig.DEFAULT_FIXED_SEQUENCE, num_token_to_add=token_to_add
+        )
+
```

Comment:
You can retrieve those values with:

```suggestion
batch_size, seq_length = input_dict["input_ids"].shape
```

That way you do not have to "copy" this part from `generate_dummy_inputs`.